src/index.js

```diff
@@ -13,9 +19,13 @@ export {
   actions,
   actionTypes,
   saga,
+  reducerHelpers,
   isAsyncOperation,
   isAsyncComplete,
-  isAsyncFailure
+  isAsyncFailure,
+  reducer,
+  selectors,
+  STORE_DOMAIN
 }
 
 export default {
```

Comment:
This isn't your newly added code, but I'm curious.  Why do we export and export default?